src/main/resources/META-INF/spring/applicationContext.xml

```diff
@@ -9,7 +9,16 @@
 
     <import resource="classpath:/META-INF/spring/projectName-domain.xml" />
 
-    <bean id="passwordEncoder" class="org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder" />
+    <bean id="passwordEncoder" class="org.springframework.security.crypto.password.DelegatingPasswordEncoder">
+        <constructor-arg name="idForEncode" value="pbkdf2" />
+        <constructor-arg name="idToPasswordEncoder">
+            <map>
+                <entry key="pbkdf2">
+                    <bean class="org.springframework.security.crypto.password.Pbkdf2PasswordEncoder" />
+                </entry>
```

Comment:
- [ ] I think better to add entry of `BCryptPasswordEncoder` etc...
This is because there is no point in using `DelegetingPasswordEncoder`.
How are you think?

Reply:
All algorithms recommended by OWASP have been fixed so that they can be used for matching.

9032183